### PR TITLE
UI test for event details section on metrics page

### LINKED ISSUE #644: Cypress: Verify event details section on Metrics page
**Purpose**
After sending a request to the `/api/track/` endpoint, verify that the event details section of the Metrics page is populated with a new event that has correct data

**Test steps**
1. Login
2. Navigate to the Settings -> Developer Settings page
3. Click the "Add API Key" button
4. In the "Create API Key" modal, enter text for the "API Key Name" field
5. In the "Create API Key" modal, select a date for the "Expiry Date + Time" field
6. In the "Create API Key" modal, click the "Confirm" button
7. In the "New API Key" modal, copy the API key that is displayed
8. Send a request to the `/api/track/` endpoint using the API key that was generated
9. Navigate to the Metrics page
10. Verify that the event details section displays a new event with the following fields: `customer_id`, `event_name`, `ID`, `time_created`
11. Verify fields of newly created event match the data that was supplied in the request

**Notes**
Cypress tests can be executed by running the `npx cypress run` command inside the `frontend` directory.
Function for sending HTTP requests: https://docs.cypress.io/api/commands/request

### PR DESCRIPTION
**Pull request by djanda97**
Fixes #644

### CODE REVIEW

File: frontend/cypress/e2e/spec.cy.ts [hunk 1]
```diff
@@ -82,3 +82,57 @@ describe("Testing Create Customer and Attach Subscription", () => {
...
+        }
+      });
+    });
+    cy.visit("http://localhost:3000/metrics");
+    cy.contains("Create Metric");
+    cy.get(".ant-collapse-header").first().click();
+    cy.contains("event_name").siblings().should("include.text", eventName);
+    cy.contains("customer_id").siblings().should("include.text", customerId);
+    cy.contains("ID").siblings().should("include.text", idempotencyId);
+    const dateString = date.toLocaleString("en-ZA").replace(",", "");
```
Comment (djanda97) [inline frontend/cypress/e2e/spec.cy.ts:134]: The `en-ZA` locale seemed to match the date format displayed in the UI the closest, but I can change this if there are better options.

**Comment by djanda97** [comment]
Some of the locators/selectors I chose probably could have been better, it has been a while since I've used Cypress 😆

**Comment by mnida** [comment]
Gonna start referring to you as dgoatda @djanda97

File: frontend/cypress/e2e/spec.cy.ts (hunk 1 above)
Comment (mnida) [reply frontend/cypress/e2e/spec.cy.ts:135]: should be fine here

**Review by mnida — APPROVED** [review]

**Pull request merged**